archetype-models/archetype-descriptor/pom.xml

```diff
@@ -39,7 +39,9 @@
         <groupId>org.codehaus.modello</groupId>
         <artifactId>modello-maven-plugin</artifactId>
         <configuration>
-          <version>1.1.0</version>
+          <!-- always show a since -->
+          <firstVersion>0</firstVersion>
```

Comment:
Where does this element come from? I don't see it either in the XSD nor the [POM reference](https://maven.apache.org/pom.html#Plugins)?

Reply:
it is configuration option for modello plugin

https://codehaus-plexus.github.io/modello/modello-maven-plugin/xdoc-mojo.html#firstVersion

Reply:
I would like to see `since` always

Reply:
Ah okay, newer saw this before :)